WonderPosition.py

```diff
@@ -0,0 +1,19 @@
+class Position(object):
+    def __init__(self,x,y,z):
+        self.X = x
```

Comment:
Ich würde mich an den Konventionen halten und 
```suggestion
        self.x = x
```
schreiben. Wenn man großes `X` verwendet würde ich erstmal erwarten, dass es sich dabei um eine Klasse und keine Variable handelt.